### ISSUE TITLE
Ant Design `ConfigProvider` does not override classes

### ISSUE THREAD

**Issue by ferretwithaberet** [rjsf-team/react-jsonschema-form#2838]
### Prerequisites

- [X] I have searched the existing issues
- [X] I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.
- [X] I have read the [documentation](https://react-jsonschema-form.readthedocs.io/)
- [ ] Ideally, I'm providing a [sample JSFiddle](https://jsfiddle.net/n1k0/f2y3fq7L/6/) or a [shared playground link](https://rjsf-team.github.io/react-jsonschema-form/) demonstrating the issue.

### What theme are you using?

antd

### Version

4.1.1

### Current Behavior

Ant Design's `ConfigProvider` does not override classes of the components used by `react-jsonschema-form` using `prefixCls` prop.

### Expected Behavior

Ant Design's `ConfigProvider` does override classes of the components used by `react-jsonschema-form` using `prefixCls` prop.

### Steps To Reproduce

1. Create react app.
2. Install `@rjsf/antd` and it's perequisites.
3. Add `ConfigProvider` to `App.jsx` with a value for the `prefixCls` prop.
4. Place the `Form` somewhere inside `ConfigProvider`.
5. Check if classes used by ant design are prefixed with the value of `prefixCls`.

### Environment

```markdown
- OS: EndeavourOS Linux x86_64 (Kernel 5.17.4-arch1-1 )
- Node: 16.14.0
- npm: 8.3.1
```

**Comment by heath-freenome** [comment]
@ferretwithaberet Can you confirm whether this is a problem with the 5.x version? Also, providing a codesandbox.io or JSFiddle with a working example would be most helpful

**Comment by ferretwithaberet** [comment]
> @ferretwithaberet Can you confirm whether this is a problem with the 5.x version?

Seems to work with v5.x.

**Comment by miguelgrc** [comment]
Hi, I still have this issue with the latest version of `@rjsf/antd`. Here is a simple [example](https://codesandbox.io/s/react-jsonschema-form-antd-forked-d6yc3l?file=/index.js) to demonstrate. 

Notice how the `prefixCls` doesn't apply to the input added by rjsf while it does for the second input, which is a normal antd input. Also, the `colorPrimary` makes the second input have a red border on focus while the first input has the default light blue border.

Notice also how the rjsf-generated elements have the `.css-dev-only-do-not-override-nllxry` class while the other input has `.css-dev-only-do-not-override-e4x42x`. The hashes are different, which explains why the styles are not being applied to one of them.

Could you please have a look? Thank you.

**Comment by dllnaoki** [comment]
have you solved this problem? @miguelgrc

**Comment by miguelgrc** [comment]
@dllnaoki, sorry for the late response. It seems to have been fixed in `@rjsf/antd` v5.18.0. I believe by https://github.com/rjsf-team/react-jsonschema-form/pull/4132
